src/invocation/ClusterService.ts

```diff
@@ -161,6 +162,14 @@ export class ClusterService extends EventEmitter {
         return this.ownerConnection;
     }
 
+    getConnectedServerVersion(): number {
```

Comment:
each connected server may have a different version, are you returning the connection for the owner member? Or some other member? Why is there a for loop here?

Reply:
I think the original method is flawed. This should look for the owner connection's version.